packages/expandable/src/component.tsx

```diff
@@ -51,13 +51,12 @@ export function Expandable(props: ExpandableProps) {
           aria-expanded={stateExpanded}
           className={classNames({
             [buttonClass || '']: true,
-            [buttonReset + ' hover:underline focus:underline']: true,
-            ['w-full text-left relative ' + boxClasses.box]: box,
-            'hover:text-aqua-700 active:text-aqua-800': info,
+            [ccExpandable.button]: true,
+            [ccExpandable.buttonBox]: box,
           })}
           onClick={() => toggleExpandable(stateExpanded)}
         >
-          <div className="flex justify-between align-center">
+          <div className='flex justify-between align-center'>
```

Comment:
Shouldn't this be double quotes? Or is this some kind of React thingie that I am not familiar with? 😅

Reply:
noo it's the same thing! hm prettier should have fixed that, will look at it and change 😅